src/features/pages/home/our-platforms/data.tsx

```diff
@@ -157,24 +157,15 @@ const platformSliderItems: SmartPlatformItem[] = [
                     />,
                 ],
             },
-            image: ({ is_eu }) =>
-                is_eu ? (
-                    <StaticImage
-                        objectFit="contain"
-                        src="../../../../images/common/home/rebranding/platform_deriv_trader_eu.png"
-                        alt="deriv trader eu"
-                        formats={['avif', 'webp', 'auto']}
-                        placeholder="none"
-                    />
-                ) : (
-                    <StaticImage
-                        objectFit="contain"
-                        src="../../../../images/common/home/rebranding/platform_deriv_trader.png"
-                        alt="deriv trader"
-                        formats={['avif', 'webp', 'auto']}
-                        placeholder="none"
-                    />
-                ),
+            image: (
+                <StaticImage
+                    objectFit="contain"
+                    src="../../../../images/common/home/rebranding/platform_deriv_trader.png"
+                    alt="deriv trader"
+                    formats={['avif', 'webp', 'auto']}
+                    placeholder="none"
```

Comment:
the image size is not ok, it should be exactly : 1824px * 1244px, please ask the designer to update it. the image size and style should be exactly like other platforms.